x-pack/plugins/security_solution/public/timelines/components/timeline/kpi/kpis.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useMemo } from 'react';
+import styled from 'styled-components';
+
+import { EuiFlexItem, EuiFlexGroup, EuiToolTip, EuiBadge } from '@elastic/eui';
+import numeral from '@elastic/numeral';
+import { euiThemeVars } from '@kbn/ui-theme';
+import { DEFAULT_NUMBER_FORMAT } from '../../../../../common/constants';
+import { useUiSetting$ } from '../../../../common/lib/kibana';
+import type { TimelineKpiStrategyResponse } from '../../../../../common/search_strategy';
+import { getEmptyValue } from '../../../../common/components/empty_value';
+import * as i18n from './translations';
+
+export const StatsContainer = styled.span`
+  font-size: ${euiThemeVars.euiFontSizeXS};
+  font-weight: ${euiThemeVars.euiFontWeightSemiBold};
+  /* border-right: ${euiThemeVars.euiBorderThin}; */
+  padding-right: 16px;
+  .smallDot {
+    width: 3px !important;
+    display: inline-block;
+  }
+  .euiBadge__text {
+    text-align: center;
+    width: 100%;
+  }
+`;
+
+export const TimelineKPIs = React.memo(({ kpis }: { kpis: TimelineKpiStrategyResponse | null }) => {
+  const kpiFormat = '0,0.[000]a';
+  const [defaultNumberFormat] = useUiSetting$<string>(DEFAULT_NUMBER_FORMAT);
+  const formattedKpis = useMemo(() => {
+    return {
+      process: kpis === null ? getEmptyValue() : numeral(kpis.processCount).format(kpiFormat),
+      user: kpis === null ? getEmptyValue() : numeral(kpis.userCount).format(kpiFormat),
+      host: kpis === null ? getEmptyValue() : numeral(kpis.hostCount).format(kpiFormat),
+      sourceIp: kpis === null ? getEmptyValue() : numeral(kpis.sourceIpCount).format(kpiFormat),
+      destinationIp:
+        kpis === null ? getEmptyValue() : numeral(kpis.destinationIpCount).format(kpiFormat),
+    };
+  }, [kpis]);
+
+  const formattedKpiToolTips = useMemo(() => {
+    return {
+      process: numeral(kpis?.processCount).format(defaultNumberFormat),
+      user: numeral(kpis?.userCount).format(defaultNumberFormat),
+      host: numeral(kpis?.hostCount).format(defaultNumberFormat),
+      sourceIp: numeral(kpis?.sourceIpCount).format(defaultNumberFormat),
+      destinationIp: numeral(kpis?.destinationIpCount).format(defaultNumberFormat),
+    };
+  }, [kpis, defaultNumberFormat]);
+
+  const getColor = useCallback((count) => {
+    if (count === 0) {
+      return 'hollow';
+    }
+    return 'hollow';
+  }, []);
+
+  return (
+    <EuiFlexGroup wrap data-test-subj="siem-timeline-kpis">
```

Comment:
I think we should add `responsive={false}` here to make the UI look like this on a small screen
![Screenshot 2023-11-15 at 3 33 05 PM](https://github.com/elastic/kibana/assets/17276605/29cd9f8e-9de5-4175-97b4-67dfb330163c)

instead of like this
![Screenshot 2023-11-15 at 3 35 24 PM](https://github.com/elastic/kibana/assets/17276605/2b783b36-4598-464c-a73c-f59cb0e53271)